src/issues/util.ts

```diff
@@ -492,10 +492,19 @@ export async function createGithubPermalink(
 		}
 	}
 	const pathSegment = uri.path.substring(repository.rootUri.path.length);
-	const rangeString = range ? `#L${range.start.line + 1}-L${range.end.line + 1}` : '';
+	const rangeString = () => {
+		if (!range) {
+			return '';
+		}
+		let hash = `#L${range.start.line + 1}`;
+		if (range.start.line === range.end.line) {
```

Comment:
Should be `!==` instead of `===`?

Reply:
🤦‍♀️ That's even what I wrote above!
```suggestion
		if (range.start.line !== range.end.line) {
```

Reply:
Thanks!